### PR TITLE
fixed longitudinal bug

### PR DESCRIPTION
**Pull request by ashih2018**
_Pull requests into Flux require the following. Submitter should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable (**N/A**) and :white_check_mark:. Reviewers should complete every item on the bulleted list when reviewing._

Fixed longitudinal table visualizer bug as noted in JIRA task #2122. This error occurs when there is no lab data for the labs, so I changed it so that it displays None when there are no labs.
I also changed the css for the banded graph visualizer when it displays none so that it uses the css class name that is used in tabular list visualizer when there are no entries. However, there is a line under the subsection name in tabular list that is not in banded graph so it looks weird right now.

Potential shortcomings:
Right now, when there is no data for a section, I skip it but we should in the future maybe look at making an empty table row when there are other subsections involved. For example, if we have white blood cells and hemoglobin lab data and there is no lab data in white blood cells, rather than showing 'no data' for white blood cells, the entire row is skipped and the table will only show the data for hemoglobin, which may cause a misunderstanding of information for the clinician (not having lab data vs not having the subsection at all)

## Submitter:

**Running the Application**

- [x] Manually tested in Chrome (primary browser for testing)
- [x] Manually tested in IE/Safari (verify app loads and basic feature testing)
- [x] Manually tested in Chrome in iPad mode (at minimum, viewing patient record)

**Task**

- [x] Branch implements task as expected
- [x] Key application features are updated as needed for task and tested (see: [Key Application Features](https://github.com/FluxNotes/flux/wiki/Key-Application-Features))
- [x] Task specific definition of done (as described in the Jira task) is met, if applicable.

**Documentation**

- [x] This pull request describes why these changes were made
- [x] Recognizes any potential shortcomings/bugs in the description 
- [ ] Pre release script is updated on the [Pre Release Testing Script wiki page](https://github.com/FluxNotes/flux/wiki/Pre-Release-Testing-Script) (add/remove single features to test, do not run through full script)
- [ ] Update the list of key features with new features on the [Key Application Features wiki page](https://github.com/FluxNotes/flux/wiki/Key-Application-Features)
- [ ] Document any new APIs/extension points
- [ ] Additional technical components and libraries are documented and added to [wiki](https://github.com/FluxNotes/flux/wiki/About-Flux-Notes) as needed

**Code Quality**

- [X] Code style guide is followed (see: [Code Style Guide](https://github.com/FluxNotes/flux/wiki/JavaScript-and-HTML-Code-Style-Guide))
- [X] Code is commented

**Tests**

- [X] Existing tests passed
- [ ] Added backend tests for new functionality added

## Reviewers:

- Check that code is maintainable and reusable. Code reuses existing code and infrastructure where appropriate.
- Check the PR and code accomplishes the task's purpose.
- Check that new tests appropriately test the new code, including edge cases. Other existing tests pass.
- Verify Pull Request checklist is correctly completed by submitter.
- Try to break the code

### CODE REVIEW

**Comment by danlee1025** [comment]
This looks good, needs a rebase.

File: src/summary/LongitudinalTableVisualizer.jsx [hunk 1]
```diff
@@ -22,7 +22,8 @@ export default class LongitudinalTableVisualizer extends Visualizer {
             favoritedNames.push(...this.props.preferenceManager.getPreference(name));
         }
         for (let conditionIndex = 0; conditionIndex < section.length; conditionIndex++) {
-            if (section[conditionIndex].data_cache) {
+            const data = section[conditionIndex].data_cache;
+            if (data && data.length > 0) {
```
[the remark this answers is not in the document]
Comment (ashih2018) [reply src/summary/LongitudinalTableVisualizer.jsx]: Thank you Keeyan Ghoreshi!

**Comment by ashih2018** [comment]
Reabsed!

**Comment by jafeltra** [comment]
I had mostly reviewed this and it looks good to me! But I'll let Dan be the official second approval since he had looked at it earlier.

**Pull request merged**